website/docs/migration.mdx

```diff
@@ -441,6 +441,46 @@ package which automatically configures `InitializeColorMode` along with
   `FormErrorMessage` won't be visible. The only way to make it visible is by
   passing `isInvalid` and setting it to `true`.
 
+### Icons
+
+## Changes
+
+- Basic interface icons provided by Chakra have moved to the `icons` package.
+  Replace all `<Icon name="..." />` elements imported from `core` with
+  equivalent _named_ icon React elements imported from `icons`.
+
+  > Replacement logic: If `<Icon name="search" />` is used, then replace it with
+  > `<SearchIcon />` from Chakra icons package.
+
+```diff
+- import { Icon } from "@chakra-ui/core"
+- <Icon name="search" />
++ import { SearchIcon } from "@chakra-ui/icons"
++ <SearchIcon />
+```
+
```

Comment:
Here we introduce the Icons package which isn't really discussed anywhere outside Buttons (if at all) and show how to use it to upgrade.